modules/history/src/views/full/conversation.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import style from './style.scss'
+
+import 'react-day-picker/lib/style.css'
+
+import { TiRefresh } from 'react-icons/ti'
+import { FaFilter } from 'react-icons/fa'
+import DayPickerInput from 'react-day-picker/DayPickerInput'
+
+function QueryOptions(props) {
+  return (
+    <div className={style['query-options']}>
```

Comment:
Maybe only a personal preference, but when using sass, I like to use camel casing for styling, so it can be written without brackets (eg: `style.queryOptions` instead of `style['query-options']`), which I find that it makes it more readable when there is a lot of them